stable/mongodb-replicaset/README.md

```diff
@@ -91,6 +91,7 @@ The following table lists the configurable parameters of the mongodb chart and t
 | `auth.metricsPassword`              | MongoDB clusterMonitor password                                           | ``                                                  |
 | `auth.existingMetricsSecret`        | If set, and existing secret with this name is used for the metrics user   | ``                                                  |
 | `auth.existingAdminSecret`          | If set, and existing secret with this name is used for the admin user     | ``                                                  |
+| `auth.secretAnnotations`            | Annotations to be added to the secret if auth is enabled                  | `{}`                                                |
```

Comment:
Would prefer just `secretAnnotations` and not be nested under auth. This is more inline with how we have `podAnnotations`, `secretAnnotations`, etc. Besides that looks good.

Reply:
Thanks so much @steven-sheehy taking the time to review the PR, I've updated the PR with the recommended changes here https://github.com/helm/charts/pull/21704/commits/26ba742b2bc928e487bedbf5171bb3e3a0c58793